pallets/amm/src/benchmarking.rs

```diff
@@ -10,21 +10,20 @@ use frame_benchmarking::{
 };
 use frame_support::assert_ok;
 use frame_system::{self, RawOrigin as SystemOrigin};
-use orml_traits::MultiCurrency;
 use primitives::TokenSymbol;
 use sp_std::prelude::*;
 
-const BASE_ASSET: CurrencyId = CurrencyId::Token(TokenSymbol::xDOT);
-const QUOTE_ASSET: CurrencyId = CurrencyId::Token(TokenSymbol::DOT);
+const BASE_ASSET: CurrencyOrAsset = CurrencyOrAsset::NativeCurrency(TokenSymbol::xDOT);
```

Comment:
Can we change the name? `CurrencyOrAsset` is too ugly

Reply:
@GopherJ  I added as per this notion page https://www.notion.so/parallelfinance/Currency-and-Assets-Adapter-b6c5995794574086b40079ae2947dce6

@ETeissonniere Could you please suggest some other name?

Reply:
@GopherJ let us know which name you would prefer :)